### DISCUSSION TITLE
Giving motion to bodies

### DISCUSSION THREAD

**Discussion by nimeeshrathi** [InteractiveComputerGraphics/SPlisHSPlasH#281] [Q&A] [answered]
Thanks Jan for this wonderful software ,
I want to do simulation from t =0 to 30 sec and create a motion of  opening of car decklid from 19 to 22 sec . Any possible solution how I can rotate the decklid from certain time step ?
Thanks in advance .

**Comment by janbender** [comment, marked as answer]
Either you could use an angular motor and use the field "targetSequence" to implement this motion. The field contains a sequence of time,target values. Similar to example "MotorScene2.json". However, I would use a "TargetAngleMotorHingeJoints".

Or you could use our Python binding. E.g., by using an embedded Python script (see AnimatedBody_2D.json/py).

**Reply by nimeeshrathi** [reply]
Thanks for your reply 
I will work on it
